core/controllers/voice_artist_test.py

```diff
@@ -416,3 +418,188 @@ def test_cannot_deassign_random_user_from_voice_artist(self):
         self.assertEqual(
             response['error'], 'Sorry, we could not find the specified user.')
         self.logout()
+
+
+class AudioUploaderHandlerTests(BaseVoiceArtistControllerTests):
```

Comment:
Please add a comment under the class to explain what is being tested.
e.g.
https://github.com/oppia/oppia/blob/5526de9cb9b9053d49799b53221f723c849310ac/core/controllers/editor_test.py#L1283-L1284